doc/user-manual/language/safe-agda.lagda.rst

```diff
@@ -51,22 +51,25 @@ then all its imported modules must also be declared safe.
 
 .. NOTE::
 
-   The ``--safe`` option turns off the ``--guardedness`` and
-   ``--sized-types`` options that are both on by default. One of them
-   (but not both, see above) can be turned on again by giving either
-   ``--guardedness`` or ``--sized-types`` *after* the  ``--safe`` option, e.g.
+   The ``--guardedness`` and ``--sized-types`` options are both on by default.
+   Setting the ``--safe`` option will ignore these defaults and only work if at most
+   one of them is explicitly turned on.
 
    .. code-block:: agda
 
      {-# OPTIONS --safe --guardedness #-}
 
-  will turn on ``--safe``, ``--no-sized-types`` (implied by ``--safe``) and
-  ``--guardedness``, while
+  will turn on ``--safe``, ``--no-sized-types`` and ``no-guardedness`` (implied by ``--safe``)
+  and then set ``--guardedness``, while
 
    .. code-block:: agda
 
      {-# OPTIONS --guardedness --safe #-}
 
-  will turn on ``--safe``, ``--no-sized-types`` (implied by ``--safe``) and
-  ``--no-guardedness`` (implied by ``--safe``; the earlier option
-  ``--guardedness`` has been overruled by ``--safe``).
+  will turn on ``guardedness``, then ``--safe`` and ``--no-sized-types`` (implied by ``--safe``).
```

Comment:
Isn't it more useful to point out that this gives the same result? E.g.
```
Hence

    .. code-block:: agda

      {-# OPTIONS --safe --guardedness #-}

will turn on ``--safe``, ``--no-sized-types`` (implied by ``--safe``), and ``--no-guardedness``
(implied by ``--safe``), while either of

    .. code-block:: agda

      {-# OPTIONS --safe #-}

or

    .. code-block:: agda

      {-# OPTIONS --guardedness --safe #-}

will turn on ``--safe``, ``--no-sized-types`` (implied by ``--safe``), and `--guardedness``. Finally e.g.

    .. code-block:: agda

      {-# OPTIONS --guardedness --safe --sized-types #-}

will fail because both ``--guardedness`` and ``--sized-types`` are set together with ``--safe``.
```

Reply:
The first one does result in `--guardedness` being turned on because that is
what the user requested.

The second one will result in `--no-guardedness`.

Reply:
Sorry, I messed up my copy-pasting! I meant block to be `--safe`, the second to be `--safe --guardedness`, and the third ``--guardedness --safe`` (as it was).